hooks/environment

```diff
@@ -90,8 +86,8 @@ if [[ -n "$vault_server" ]] ; then
     "${vault_path}"
     "${VAULT_BASE_PATH}"
   )
-
   secrets=''
+  # shellcheck disable=SC2048
```

Comment:
this should not be disabled as it is an actual possible bug. In particular, vault paths can have spaces in them and as we can't assume anything about the plugin's configuration you should make sure that those paths are respected as-is.

Reply:
Done, that makes sense!